public/app/features/alerting/unified/components/rules/EditRuleGroupModal.tsx

```diff
@@ -64,35 +243,86 @@ export function EditCloudGroupModal(props: ModalProps): React.ReactElement {
     );
   };
 
+  const formAPI = useForm<FormValues>({
+    mode: 'onSubmit',
+    defaultValues,
+    shouldFocusError: true,
+  });
+  const {
+    handleSubmit,
+    register,
+    watch,
+    formState: { isDirty, errors },
+  } = formAPI;
+
+  const onInvalid = () => {
+    notifyApp.error('There are errors in the form. Please correct them and try again!');
+  };
+
+  const rulerRuleRequests = useUnifiedAlertingSelector((state) => state.rulerRules);
+  const groupfoldersForSource = rulerRuleRequests[getRulesSourceName(namespace.rulesSource)];
+
   return (
     <Modal
       className={styles.modal}
       isOpen={true}
-      title="Edit namespace or rule group"
+      title="Edit namespace or evaluation group"
       onDismiss={onClose}
       onClickBackdrop={onClose}
     >
-      <Form defaultValues={defaultValues} onSubmit={onSubmit} key={JSON.stringify(defaultValues)}>
-        {({ register, errors, formState: { isDirty }, watch }) => (
+      <FormProvider {...formAPI}>
+        <form onSubmit={(e) => e.preventDefault()} key={JSON.stringify(defaultValues)}>
           <>
-            <Field label="Namespace" invalid={!!errors.namespaceName} error={errors.namespaceName?.message}>
+            <Field
+              label={
+                <Label htmlFor="namespaceName">
+                  <Stack gap={0.5}>
+                    NameSpace
+                    <InfoIcon text={'You can update name space'} />
+                  </Stack>
+                </Label>
+              }
+              invalid={!!errors.namespaceName}
+              error={errors.namespaceName?.message}
+            >
               <Input
                 id="namespaceName"
                 {...register('namespaceName', {
                   required: 'Namespace name is required.',
                 })}
               />
             </Field>
-            <Field label="Rule group" invalid={!!errors.groupName} error={errors.groupName?.message}>
+            <Field
+              label={
+                <Label htmlFor="groupName">
+                  <Stack gap={0.5}>
+                    Evaluation group
+                    <InfoIcon text={'You can update group name'} />
+                  </Stack>
+                </Label>
+              }
+              invalid={!!errors.groupName}
+              error={errors.groupName?.message}
+            >
               <Input
                 id="groupName"
                 {...register('groupName', {
-                  required: 'Rule group name is required.',
+                  required: 'Evaluation group name is required.',
                 })}
               />
             </Field>
             <Field
-              label="Rule group evaluation interval"
+              label={
+                <Label
+                  htmlFor="groupInterval"
+                  description="Evaluation interval should be smaller or equal than For values for existing rules in this group."
+                >
```

Comment:
same as above

Reply:
Maybe written this way makes more sense: 
Evaluation interval should be smaller or equal than `For` values for existing rules in this group.